test/tokenizer_test.dart

```diff
@@ -98,7 +103,15 @@ class TokenizerTestParser {
     // TODO(jmesserly): when debugging test failures it can be useful to add
     // logging here like `print('ParseError $token');`. It would be nice to
     // use the actual logging library.
-    outputTokens.add(["ParseError", token.data]);
+    addOutputToken(token, ["ParseError", token.data]);
+  }
+
+  void addOutputToken(Token token, List array) {
+    outputTokens.add([
+      ...array,
+      if (token.span != null && _generateSpans) token.span.start.offset,
+      if (token.span != null && _generateSpans) token.span.end.offset,
+    ]);
```

Comment:
cc @kevmoo 

The spread and if elements require an SDK version of 2.2.2 or above so this is breaking some of the travis ci bots. If you are going to use the features it will require updating the minimum bound on the SDK in the pubspec.yaml and the .travis.yml.

https://github.com/dart-lang/html/blob/d37f5887e327a245abb9fc3675446dd96b172957/pubspec.yaml#L9

https://github.com/dart-lang/html/blob/d37f5887e327a245abb9fc3675446dd96b172957/.travis.yml#L5

Reply:
Done.

Reply:
@nshahan I've added an entry to the CHANGELOG.